### DISCUSSION TITLE
How to increase loading speed of embedded links?

### DISCUSSION THREAD

**Discussion by louis030195** [excalidraw/excalidraw#6875] [Q&A]
Hey, I use excalidraw everywhere, and for example in an app in the onboarding process but the loading is quite slow so some users drop off because of it, any way to optimise the loading speed? less items?

**Comment by ad1992** [comment]
@louis030195 can you elaborate on whether you are referring to https://excalidraw.com/ or using the npm package or embeddable feature specifically ?

**Reply by louis030195** [reply]
yes, i'm paid user on app.excalidraw.com:
<img width="515" alt="image" src="https://github.com/excalidraw/excalidraw/assets/25003283/c67bc1e0-14e3-4c41-9a17-35fbe45b0f9e">

**Reply by louis030195** [reply]
<img width="571" alt="image" src="https://github.com/excalidraw/excalidraw/assets/25003283/5060d866-c094-432d-878d-fd9484ea77cf">

**Reply by dwelle** [reply]
Hi @louis030195,

Can you email us on `support@excalidraw.com` with details, or drop a message in our Discord? I'm getting very different results.

Lighthouse:

![image](https://github.com/excalidraw/excalidraw/assets/5153846/677626ef-cd1e-4a26-9be0-65265a45a3de)

GTmetrix:

![image](https://github.com/excalidraw/excalidraw/assets/5153846/35d4be2d-f982-41e1-acf8-4245c0a90d95)
